common-crafting.lic

```diff
@@ -12,26 +12,26 @@ module DRCC
     ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
   end
 
-  def find_empty_crucible
+  def find_empty_crucible(idle_room = nil, crucibles)
```

Comment:
Ahh, didn't realize that. Hometown string it is.

On Tue, Sep 6, 2016 at 2:22 PM, Robert Herbig <notifications@github.com>
wrote:

> In common-crafting.lic
> <https://github.com/rpherbig/dr-scripts/pull/751#discussion_r77690814>:
>
> > @@ -12,26 +12,26 @@ module DRCC
> >      ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
> >    end
> >
> > -  def find_empty_crucible
> > +  def find_empty_crucible(idle_room = nil, crucibles)
>
> I'm not a big fan of calling get_settings from the helpers since it won't
> account for args that were passed to the script which load other setting
> files.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/751/files/d56664e67f7772e62511263960f200f4536e4317#r77690814>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARQKjnhOy8FjLCdo4vhuWz4MUWxaGV-Qks5qna93gaJpZM4J2CdL>
> .
>